pkg/netceptor/netceptor.go

```diff
@@ -221,6 +222,8 @@ const (
 	ConnTypeStreamTLS = 2
 )
 
+var ConnTypeStrings = [...]string{"Datagram", "Stream", "StreamTLS"}
```

Comment:
what if this was a map instead?

`connTypeStrings := map[byte] string {ConnTypeDatagram: "DataGram" ... }`

then in GetConnectionTypeAsString() we do

```
conntypestr, ok := connTypeString[conntype]
if !ok {
conntypestr = "unknown"
}
```

Reply:
I like the idea of the map because then the labels are not dependent on the order of the array.